pkg/plugin/v2/scaffolds/internal/templates/kustomize.go

```diff
@@ -102,15 +102,15 @@ vars:
 #  objref:
 #    kind: Certificate
 #    group: cert-manager.io
-#    version: v1alpha2
+#    version: v1
 #    name: serving-cert # this name should match the one in certificate.yaml
 #  fieldref:
 #    fieldpath: metadata.namespace
 #- name: CERTIFICATE_NAME
 #  objref:
 #    kind: Certificate
 #    group: cert-manager.io
-#    version: v1alpha2
+#    version: v1
```

Comment:
The `pkg/plugin/v2` ought to receive only fix bugs and cannot have breaking changes. All improvements and etc needs to be addressed only  in `pkg/plugin/v3`. Could you please update it?

Reply:
See ^ we can just perform the change inside of `pkg/plugin/v3`. Could you please revert the changes on `pkg/plugin/v2` and re-run the make generate?

Reply:
Solved. Feel free to close this one.